src/fmu/sumo/explorer/_case.py

```diff
@@ -14,14 +18,38 @@ def __init__(self, sumo_client, meta_data):
 
         self.sumo_id = self.meta_data["_id"]
         self.fmu_id = source["fmu"]["case"]["uuid"]
-        self.case_name = source["fmu"]["case"]["name"]
+        self.name = source["fmu"]["case"]["name"]
         self.field_name = source["masterdata"]["smda"]["field"][0]["identifier"]
         self.status = source["_sumo"]["status"]
         self.user = source["fmu"]["case"]["user"]["id"]
         self.object_type = "case"
 
 
+    @property
+    def sumo_id(self):
+        """Returns attribute _sumo_id"""
+        return self._sumo_id
+
+    @sumo_id.setter
+    def sumo_id(self, sumo_id):
+        """Sets attribute _sumo_id
+        args:
+            sumo_id (str): a sumo_id
+        """
+        self._sumo_id = sumo_id
+
+    @property
+    def name(self):
+        """returns name attribute"""
+        return self._name
+
+    @name.setter
+    def name(self, name):
```

Comment:
Probably should not have a setter for `name`, either :-)

Reply:
Agree, fixed in commit 61bca33 on fork